spec/unit/analytics/google-analytics-universal-tracker.spec.js

```diff
@@ -100,6 +100,13 @@ describe('GOVUK.GoogleAnalyticsUniversalTracker', function () {
       )
     })
 
+    it('tracks custom dimensions', function() {
+      universal.trackEvent('category', 'action', {dimension29: 'Home'})
+      expect(window.ga.calls.mostRecent().args).toEqual(
+        ['send', {hitType: 'event', eventCategory: 'category', eventAction: 'action', dimension29: 'Home'}]
```

Comment:
Have you confirmed that this object structure correctly passes the dimension to that event hit?